controllers/apartments.rb

```diff
@@ -0,0 +1,67 @@
+get '/apartments' do
+  @apartments = Apartment.all
+  erb :"apartments/index"
+end
+
+get '/apartments/new' do
+  erb :"apartments/new"
+end
+
+get "/apartments/:id/edit" do
+  @id = params[:id]
+  @apartment = Apartment.find(@id)
+  erb :"apartments/edit"
+end
+
+get "/apartments/:id/remove" do
+  @id = params[:id]
+  @apartment = Apartment.find(@id)
+  erb :"apartments/remove"
+end
+
+get '/apartments/:id' do
+  @id = params[:id]
+  @apartment = Apartment.find(@id)
+  @tenants = @apartment.tenants
+  erb :"apartments/info"
```

Comment:
consider renaming this view to `apartments/show.erb` - this is a convention we'll use this week and next w/ rails